2-mandatory/3-financial-times.js

```diff
@@ -5,6 +5,8 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
+ 
+    return allArticleTitles.filter(items=> items.length < 65);
```

Comment:
nice use of an arrow function! although to be a bit cleaner the `items` parameter should probable be called `title` instead